worlds/witness/hints.py

```diff
@@ -2,96 +2,96 @@
 from .Options import is_option_enabled, get_option_value
 
 joke_hints = [
-    ("Quaternions", "break", "my brain"),
```

Comment:
How does the mod know if the generated seed is using the old style or the new style hints?
(i.e. does it break if you load an old seed in a new mod and vice versa?)

Reply:
New seed in old mod breaks, everything else is fine.

Reply:
Although that did just remind me that I haven't updated the required client version